src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random random = new Random();
+        Color color = ColorSupplier.getRandomColor();
```

Comment:
Move random and colorSupplier to the class field

And move 100 to the class constant. It is magic number

An example of how to do this can be seen in the checklist